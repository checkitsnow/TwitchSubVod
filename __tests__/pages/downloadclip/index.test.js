import React from 'react';
import { render, cleanup } from '@testing-library/react';
import DownloadClips from '@/pages/downloadclip';

afterEach(cleanup);

test('should render Download Clips correctly', async () => {
  const { container } = render(<DownloadClips />);
  expect(container).toMatchSnapshot();
});
