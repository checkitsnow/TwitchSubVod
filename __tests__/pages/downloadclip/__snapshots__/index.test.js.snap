// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Download Clips correctly 1`] = `
<div>
  <div
    class="DownloadClip__Container-sc-1lslxgi-0 cgprHQ"
  >
    <div
      class="DownloadClip__AnimationContainer-sc-1lslxgi-1 ffqmEF"
    >
      <h1>
        Twitch Clip Downloader
      </h1>
      <form>
        <input
          aria-label="DownloadClipInput"
          name="clip"
          placeholder="https://clips.twitch.tv/CarefulNiceJaguarRickroll"
          type="text"
          value=""
        />
        <button
          aria-label="submit"
          type="submit"
        >
          <svg
            fill="none"
            height="14"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="11"
              cy="11"
              r="8"
            />
            <line
              x1="21"
              x2="16.65"
              y1="21"
              y2="16.65"
            />
          </svg>
          Download Clip
        </button>
      </form>
      <div
        class="styles__Container-cfnpnh-0 iVQCyu"
      >
        <span>
          <svg
            fill="none"
            height="14"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="11"
              cy="11"
              r="8"
            />
            <line
              x1="21"
              x2="16.65"
              y1="21"
              y2="16.65"
            />
          </svg>
          Home
        </span>
      </div>
      <div
        class="styles__Container-cfnpnh-0 iVQCyu"
      >
        <span>
          <svg
            fill="none"
            height="14"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="11"
              cy="11"
              r="8"
            />
            <line
              x1="21"
              x2="16.65"
              y1="21"
              y2="16.65"
            />
          </svg>
          Search for deleted clips
        </span>
      </div>
      <div
        class="styles__Container-cfnpnh-0 iVQCyu"
      >
        <span>
          <svg
            fill="none"
            height="14"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="14"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="11"
              cy="11"
              r="8"
            />
            <line
              x1="21"
              x2="16.65"
              y1="21"
              y2="16.65"
            />
          </svg>
          Search for deleted vods
        </span>
      </div>
      
    </div>
    <div
      class="styles__Container-w3v8va-0 jtEnOa"
    >
      <a
        href="https://ko-fi.com/pogulive"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Buy Me a Coffee at ko-fi.com"
          class="kofi-img"
          src="https://cdn.ko-fi.com/cdn/kofi5.png?v=2"
        />
      </a>
      <a
        href="https://twitter.com/pogulive"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="twitter logo"
          src="https://abs.twimg.com/favicons/twitter.ico"
        />
      </a>
    </div>
  </div>
</div>
`;
